README.md

```diff
@@ -116,6 +116,17 @@ python setup.py install
 * [vayuda](https://github.com/vayuda) with generic bitpacking kernels that were code generated using pure PyTorch [prototype/common](torchao/prototype/common)
 * [andreaskopf](https://github.com/andreaskoepf) and [melvinebenezer](https://github.com/melvinebenezer) with [1 bit LLMs](torchao/prototype/dtypes) Bitnet 1.58 bitpacked into uint2 and fully code-generated with torch.compile
 
+## Blogs and Videos
```

Comment:
can we remove the section on "With intrusive code changes" since the content is duplicated here?

Reply:
I removed the fast blogs from here instead, I'd rather keep the instrusive code changes because I want people to understand up front that our perf benchmarks are without changing the baseline